examples/cd_clf_cifar10.ipynb

```diff
@@ -18,9 +18,17 @@
   },
```

Comment:
<p>I think this whole section is now defunct and should be removed?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi-detect/pull/206/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Reply:
Sorry I've just realised the wrong notebook is currently uploaded for some reason.. Hence the errors and things not changed. Let me just push the one that should be there

Reply:
Ok pushed now. I hadn't clicked 'save notebook' before.